app.json

```diff
@@ -0,0 +1,13 @@
+{
+  "scripts": {
+    "postdeploy": "bundle exec rails db:migrate && bundle exec rails db:seed"
+  },
+  "environments": {
+    "review": {
+      "env": {
+        "REVIEW_APP": "true"
+      },
+      "addons": ["heroku-postgresql"]
+    }
+  }
+}
```

Comment:
Need to set this on `staging` and `heroku` review apps. This ensures that the seed only runs in productino env. We _could_ define `heroku` and `staging` environments explicitly, as well (define `config/staging` `config/heroku` and then add `RAILS_ENV` for each of these environments). I think this change keeps things as "same as possible" between heroku review apps + staging and also makes `production` be fairly represented in `heroku` and `staging` unless there are huge reasons we need to actually split them out more like `vita-min`.
1. heroku: set on `app.json`
2. staging: set on pya module